cmake/defaults/msvcdefaults.cmake

```diff
@@ -99,6 +99,11 @@ if (NOT Boost_USE_STATIC_LIBS)
     _add_define("BOOST_ALL_DYN_LINK")
 endif()
 
+if(${PXR_DEFINE_BOOST_DEBUG_PYTHON_FLAG})
+    _add_define(BOOST_DEBUG_PYTHON)
+    _add_define(BOOST_LINKING_PYTHON)
```

Comment:
Nitpick: could we quote these defines to match the others in this file?

Reply:
Ah yes for sure. Sorry I missed that.